src/components/pages/AdminMeetingsDashBoard/CrudModals/CreateModal.js

```diff
@@ -0,0 +1,212 @@
+import React, { useEffect, useState } from 'react';
+import 'antd/dist/antd.css';
+import { Button, Modal } from 'antd';
+import axios from 'axios';
+
+import DynamicDropdown from '../DynamicDropdown';
+
+const CreateModal = props => {
+  const { data, setMeetings, meetings } = props;
```

Comment:
inside the function should invoke useAxiosWithAuth0 and save the instance to a variable, as following:

const axiosWithAuth = useAxiosWithAuth0()

Reply:
This has been done in the last commit. Thank you!